sql/core/src/test/resources/sql-tests/inputs/group-by-filter.sql

```diff
@@ -0,0 +1,150 @@
+-- Test filter clause for aggregate expression.
+
+-- Test data.
+CREATE OR REPLACE TEMPORARY VIEW testData AS SELECT * FROM VALUES
+(1, 1), (1, 2), (2, 1), (2, 2), (3, 1), (3, 2), (null, 1), (3, null), (null, null)
```

Comment:
This table seems to be designed for `null` value tests. Shall we have more null values in `EMP` table together?

Reply:
I think it's OK. Do you have other idea?

Reply:
My idea was to have more values in `EMP` table like `testData` table. For example, one null per column?

Reply:
Do you mean I insert some null values to ensure that each column has a null?